src/CustomElements/v1/CustomElements.js

```diff
@@ -103,7 +103,7 @@ var CustomElementDefinition;
 
     // HTML spec part 4.13.4
     // https://html.spec.whatwg.org/multipage/scripting.html#dom-customelementsregistry-define
-    define: function(name, constructor, options) {
+    define: function(name, constructor, options = {}) {
```

Comment:
I think this is probably `null` in the spec, it just makes using it simpler. Not attached to keeping the default value if spec consistency here is super important.